modules/programs/opencode.nix

```
@@ -137,6 +145,15 @@ in
       '';
     };
 
+    themes = mkOption {
+      inherit (jsonFormat) type;
```

I would also prefer if one could provide either text or a path to a file.


```suggestion
      type = lib.types.attrsOf (lib.types.either jsonFormat lib.types.path);
```